src/components/atoms/Cv/Cv.components.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
```

Comment:
To potrzebuje całkiem innej nazwy. W atomach nie powinniśmy sugerować użycia, czyli `Cv.Heading` będzie używany w innych miejscach i może to wyglądać dosyć dziwnie.

Reply:
to będę wdzięczna za propozycję, bo niczego sensownego nie wymyśliłam ;p